features/all-positions/AllLiquidations.tsx

```diff
@@ -0,0 +1,330 @@
+  &:hover {
+    cursor: pointer;
+    opacity: 0.6;
+  }
+  text-align: end;
+  user-select: none;
+`;
+
+const PageButtons = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 2em;
+  margin-bottom: 2em;
+`;
+
+type FadedDiv = {
+  faded: boolean;
+};
+
+const Arrow = styled.div<FadedDiv>`
+  color: ${({ theme }) => theme.primary1};
+  opacity: ${(props) => (props.faded ? 0.3 : 1)};
+  padding: 0 20px;
+  user-select: none;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+enum SORT_FIELD {
+  MAX_DISPUTABLE_PRICE,
+  LOCKED_COLLATERAL,
+  TOKENS_LIQUIDATED,
+  LIQUIDATION_TIMESTAMP,
+}
+
+const FIELD_TO_VALUE: { [sortField: number]: string } = {
+  [SORT_FIELD.MAX_DISPUTABLE_PRICE]: "maxDisputablePrice",
+  [SORT_FIELD.LOCKED_COLLATERAL]: "lockedCollateral",
+  [SORT_FIELD.TOKENS_LIQUIDATED]: "tokensLiquidated",
+  [SORT_FIELD.LIQUIDATION_TIMESTAMP]: "liquidationTimestamp",
+};
+
+const ITEMS_PER_PAGE = 10;
+
+const AllLiquidations = () => {
+  const { empState } = EmpState.useContainer();
+  const { priceIdentifier: priceId, liquidationLiveness } = empState;
+  const { symbol: tokenSymbol } = Token.useContainer();
+  const { symbol: collSymbol } = Collateral.useContainer();
+  const { liquidations } = EmpSponsors.useContainer();
+  const { getEtherscanUrl } = Etherscan.useContainer();
+
+  // Pagination
+  const [page, setPage] = useState<number>(1);
+  const [maxPage, setMaxPage] = useState<number>(1);
+
+  // Sorting
+  const [sortDirection, setSortDirection] = useState<boolean>(true);
+  const [sortedColumn, setSortedColumn] = useState<number>(
+    SORT_FIELD.TOKENS_LIQUIDATED
+  );
+
+  // Set max page depending on # of liqs
+  useEffect(() => {
+    setMaxPage(1);
+    setPage(1);
+
+    if (liquidations) {
+      let extraPages = 1;
+      if (Object.keys(liquidations).length % ITEMS_PER_PAGE === 0) {
+        extraPages = 0;
+      }
+      setMaxPage(
+        Math.floor(Object.keys(liquidations).length / ITEMS_PER_PAGE) +
+          extraPages
+      );
+      // This will set maxPage to 0 if there are no liqs.
+    }
+  }, [liquidations]);
+
+  if (
+    tokenSymbol !== null &&
+    collSymbol !== null &&
+    priceId !== null &&
+    liquidations &&
+    liquidationLiveness !== null &&
+    Object.keys(liquidations).length > 0
+  ) {
+    const prettyBalance = (x: number) => {
+      const x_string = x.toFixed(4);
+      return utils.commify(x_string);
+    };
+
+    const prettyDate = (x_secs: number) => {
+      return new Date(x_secs * 1000).toLocaleString("en-GB", {
+        timeZone: "UTC",
+      });
+    };
+
+    const prettyAddress = (x: string) => {
+      return x.substr(0, 6) + "..." + x.substr(x.length - 6, x.length);
+    };
+
+    const invertDisputablePrice = isPricefeedInvertedFromTokenSymbol(
+      tokenSymbol
+    );
+    const getDisputablePrice = (x: string) => {
+      return invertDisputablePrice && parseFloat(x) !== 0
+        ? 1 / Number(x)
+        : Number(x);
+    };
```

Comment:
Slightly confused -- why would you need to invert the price here? Isn't the input always tokens/collateral since you're pulling the amounts directly from the liquidation? Or am I missing a step?

Reply:
The input is: 

```
parseFloat(liquidation.liquidatedCollateral) /
                    (parseFloat(liquidation.tokensLiquidated) * collReqFromWei)
```

Which sets the max price for the liquidation timestamp for it to be disputable (i.e. if the price turns out to be below this, then you can dispute the price).

For USDETH this might be 0.0028 ($359 = 1 ETH), but this dApp performs inversions on pricefeeds that are more "intuitively" read inverted such as USDETH --> ETHUSD. So, the "max disputable price" is 0.0028 but this implies a "min disputable price" of $359 (i.e. if the price is above 359 then the liquidation is disputable).

And we want to display $359 to the user instead of 0.0028.

Does this make sense?